hadoop-crypto-api/src/main/java/com/palantir/hadoop/KeyMaterials.java

```diff
@@ -98,79 +81,33 @@ public static KeyMaterial deserialize(SerializableKeyMaterial keyMaterial) {
     }
 
     public static byte[] wrap(KeyMaterial keyMaterial, PublicKey key) {
-        ByteArrayOutputStream stream = new ByteArrayOutputStream();
-        Cipher keyWrappingCipher = getCipher(Cipher.WRAP_MODE, key);
-        SecretKey secretKey = keyMaterial.getSecretKey();
-
-        try {
-            stream.write(VERSION);
-
-            String keyAlgorithm = secretKey.getAlgorithm();
-            stream.write(keyAlgorithm.length());
-            stream.write(keyAlgorithm.getBytes(StandardCharsets.UTF_8));
-
-            byte[] encryptedKey = keyWrappingCipher.wrap(secretKey);
-            stream.write(encryptedKey.length);
-            stream.write(encryptedKey);
-
-            byte[] iv = keyMaterial.getIv();
-            stream.write(iv.length);
-            stream.write(iv);
-
-            stream.close();
-            return stream.toByteArray();
-        } catch (IOException | InvalidKeyException | IllegalBlockSizeException e) {
-            throw Throwables.propagate(e);
-        }
+        return SERIALIZERS.get(KeySerializerV2.VERSION).wrap(keyMaterial, key);
```

Comment:
I need `KeySerializerV2#wrap(...)` so I can either grab it from the map or instantiate a new one. I guess I could also just have a static one in KeyMaterials? was just avoiding creating new ones (thinking in terms of something like `ObjectMappers.getMapper()`)